VisualPinball.Unity/VisualPinball.Unity/VPT/Table/TableBehavior.cs

```diff
@@ -68,6 +68,8 @@ public class TableBehavior : ItemBehavior<Engine.VPT.Table.Table, TableData>
 		// note: this cache needs to be keyed on the engine material itself so that when its recreated due to property changes the unity material
 		// will cache miss and get recreated as well
 		private readonly Dictionary<PbrMaterial, UnityEngine.Material> _unityMaterials = new Dictionary<PbrMaterial, UnityEngine.Material>();
+		// keep a list of texture names that need recreation, serialized and lazy so when undo happens they'll be considered dirty again
+		[SerializeField] private List<string> _dirtyTextures = new List<string>();

```

Comment:
`HashSet<string>` would be faster, no?

Reply:
For the access pattern yeah that would make sense, but since it needs to serialize our only collection option is List or array. In practice this isn't going to really matter anyway, the cost of iterating the texture list would be way overshadowed by loading in the real texture data.

Reply:
Oh, okay, didn't know that sets weren't serializable.